packages/bitcoin-dlc-provider/lib/BitcoinDlcProvider.ts

```diff
@@ -1984,13 +1968,16 @@ Payout Group not found',
 
     const contractId = xor(
       _dlcTransactions.fundTx.txId.serialize(),
-      dlcAccept.tempContractId,
+      dlcAccept.temporaryContractId,
     );
     _dlcTransactions.contractId = contractId;
 
     dlcAccept.cetSignatures = cetSignatures;
     dlcAccept.refundSignature = refundSignature;
-    dlcAccept.negotiationFields = new NegotiationFieldsV0();
+
+    // The following line is commented out because specifying empty
+    // negotiationFields causes dlcAccept serialization to fail
+    // dlcAccept.negotiationFields = new SingleNegotiationFields();
```

Comment:
This makes sense since `negotiationFields` was made an optional field in the `DlcAccept` in https://github.com/discreetlogcontracts/dlcspecs/pull/163/files#diff-e0f5b925f91a1c09c6daf26f9a7d28816cb9ff9f08863faca719b7ee0a1cc065R204

Looks good to remove 👍 

A future iteration of this could include the option to provide negotiation fields, but this can be added later.